homeassistant/components/comelit/light.py

```diff
@@ -49,15 +47,9 @@ def __init__(
         self._api = coordinator.api
         self._device = device
         super().__init__(coordinator)
+        self._attr_name = device.name
         self._attr_unique_id = f"{config_entry_unique_id}-{device.index}"
-        self._attr_device_info = DeviceInfo(
-            identifiers={
-                (DOMAIN, self._attr_unique_id),
-            },
-            manufacturer="Comelit",
-            model="Serial Bridge",
-            name=device.name,
-        )
+        self.coordinator.register_device()
```

Comment:
Shouldn't we register one main device when setting up the coordinator and then in the platforms, set device info in the entities that refer to the main device with `via_device`?

Reply:
Fixed